practice.html

```diff
@@ -31,6 +32,7 @@
             <div class="github-icon">
                 <a href="https://github.com/arpitghura/typing-test" class="nav-item-link"><i class='bx bx-sm bxl-github'></i></a>
             </div>
+            <button onclick="toggleLightDarkMode()" class="dark-btn">Dark Mode</button>
```

Comment:
use any icon which can be changed (like moon icon and sun icon) instead of the text `Dark Mode`. 

You can use icon from https://boxicons.com/ and the cdn for this is already linked to the files.

Reply:
Do this change.